packages/api/src/shared/logger.ts

```diff
@@ -0,0 +1,54 @@
+import winston from 'winston';
+import DailyRotateFile from 'winston-daily-rotate-file';
+
+const levels = {
+  error: 0,
+  warn: 1,
+  info: 2,
+  http: 3,
+  debug: 4,
+};
+
+const level = (): string => {
+  const env = process.env.NODE_ENV || 'development';
+  const isDevelopment = env === 'development';
+  return isDevelopment ? 'debug' : 'warn';
```

Comment:
Could we allow production log level to be specified in .ENV? For development it can always be set to debug.

Reply:
ok.. i didn't want to new LOGGING_LEVEL variable to be mandatory so i set up default level for the production also `warn`
It will log errors and warnings